Podcast/AppDelegate.swift

```diff
@@ -106,7 +107,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return false
     }
     
-
+    func logout() {
+        googleLoginViewController.logout()
+        window?.rootViewController = loginNavigationController
```

Comment:
after i logged out and logged back in again it opened to the settings page, it should probably be the feed instead

Reply:
fixed